pylsp_ruff/plugin.py

```diff
@@ -142,7 +142,7 @@ def pylsp_format_document(workspace: Workspace, document: Document) -> Generator
             # specifying `format = ["I"]` to get import sorting as part of formatting.
             new_text = run_ruff(
                 settings=PluginSettings(
-                    ignore=["ALL"],
+                    ignore=["ALL"] if "ALL" not in settings.format else [],
```

Comment:
This is admittedly a bit of a quick and dirty patch; would it be preferred to track this condition when the settings are parsed?